docker-compose.yml

```diff
@@ -48,12 +46,10 @@ services:
     command: "/bin/bash /build-docs.sh /nanoarrow"
 
   integration:
-    # Based on an arrow-dev repo image that is amd64 only
-    image: ${REPO}:integration-amd64
+    # Don't cache here (building the image takes about the same amount of time
+    # as downloading on CI)
```

Comment:
That's probably because the base image is hosted on dockerhub (and pretty big iirc), in my experience things on ghcr.io have much better download characteristics. Though maybe the build is just that fast too ^^